### DISCUSSION TITLE
Welcome to RISC-V Learn Discussions!

### DISCUSSION THREAD

**Discussion by rpsene** [riscv/learn#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  
Important: [Please ensure that you adhere to the RISC-V Code of Conduct](https://riscv.org/community/community-code-of-conduct/).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by junambi** [comment]
Hi,

I am Nambi and am an Individual member of the RISCV community.

The RISC-V community is a great place to work with and share ideas and thoughts, they are welcoming, open and supportive.

I am part of the Chairs committee and contribute to the RISCV development group.

Regards,
Nambi
